### PR TITLE
IA: Preserve Betty image crop ratio

### PR DESCRIPTION
**Pull request by mparent61**
Was previously using default "16x9" ratio. Will now pass through intended crop ratio to IA.

### CODE REVIEW

**Comment by mparent61** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;fdFxXo1P&#x2F;529-issue-with-instant-article-parsing-inline-betty-images"> Issue with Instant Article parsing inline betty images.</a>

**Comment by mparent61** [comment]
@spra85 @benghaziboy @MichaelButkovic ready for review

File: bulbs/__init__.py [hunk 1]
```diff
@@ -1 +1 @@
-__version__ = "3.9.0"
+__version__ = "3.9.1"
```
Comment (MichaelButkovic) [inline bulbs/__init__.py]: will need to bump this, since i just took 3.9.1

**Comment by MichaelButkovic** [comment]
lgtm, otherwise

File: bulbs/__init__.py (hunk 1 above)
Comment (mparent61) [inline bulbs/__init__.py]: thx

**Comment by spra85** [comment]
LGTM

**Pull request merged**